packages/inter-protocol/test/vaultFactory/test-vault-collateralization.js

```diff
@@ -0,0 +1,28 @@
+import { test as unknownTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+
+import { makeTracer } from '@agoric/internal';
+import { makeDriverContext, makeManagerDriver } from './driver.js';
+
+/** @typedef {import('./driver.js').DriverContext & {
+ * }} Context */
+/** @type {import('ava').TestFn<Context>} */
+const test = unknownTest;
+
+const trace = makeTracer('TestMCR');
+
+test.before(async t => {
+  t.context = await makeDriverContext();
+  trace(t, 'CONTEXT', t.context.rates);
+});
+
+test('excessive loan', async t => {
+  const { aeth, run } = t.context;
+  const md = await makeManagerDriver(t);
+
+  // Try to Create a loan for Alice for 5000 Minted with 100 aeth collateral
+  const vaultP = md.makeVaultDriver(aeth.make(100n), run.make(5000n));
+
+  await t.throwsAsync(vaultP, {
+    message: /Proposed debt.*5250n.*exceeds max.*476n.*for.*100n/,
+  });
```

Comment:
I like to see a test like this paired with one that passes. Bonus points when the two have parameters very close together, showing what the code is sensitive to.

Reply:
I think I covered this now. Please reopen if not